.stylelintrc.js

```diff
@@ -0,0 +1,23 @@
+module.exports = {
+  extends: [
+    'stylelint-config-standard'
+  ],
+  plugins: [
+    './lib/stylelint/expiring-todo-comments'
+  ],
+  rules: {
+    'at-rule-no-unknown': [true, {
+      ignoreAtRules: [
```

Comment:
ugh, so excited for this. looking at our css files as they are now makes me ANXIOUS